introduction/silo-entry-state.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Silo Entry State"
+icon: "key"
+description: "Understanding Silo Entry States in Invopop Workflows"
+---
+
+Let's take a step back and look at the full picture.
```

Comment:
Is this accurate? In a "full picture" I'd expect to see more things than just the silo...